sale_stock_picking_note/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    comments = fields.Text(
```

Comment:
Better `sale_comment`?

Reply:
In `sale.order`, the field for this is called `note`. In invoices, it's called `narration`, so it seems there's no consensus about this. I think `customer_note` can be a good balance. What do you think?

Reply:
Ok, I like it :)